docs/meta/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+../../CHANGELOG
```

Comment:
I've had poor experiences with using symlinks with git repos in the past, with them often times ending up broken because someone's environment (windows) doesn't have it enabled. I'd advocate for just making the `conf.py` script copy this file into place as part of the sphinx build process.

Reply:
I've followed your suggestion - thank you :hugs: